File: lwjgl3/src/main/java/io/github/mesabloo/hmdefense/lwjgl3/DesktopLauncher.java
package io.github.mesabloo.hmdefense.lwjgl3;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import io.github.mesabloo.hmdefense.app.HMMain$;

// Please note that on macOS your application needs to be started with the -XstartOnFirstThread JVM argument
public class DesktopLauncher {
    public static void main(String[] arg) {
        Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
        config.useVsync(true);
        config.setForegroundFPS(60);
        config.setTitle("Heavy Mach: Defense");
        new Lwjgl3Application(HMMain$.MODULE$, config);
    }
}

File: desktop/src/kotlin/fr/mesabloo/heavymachdefense/desktop/DesktopLauncher.kt
package fr.mesabloo.heavymachdefense.desktop

import com.badlogic.gdx.Files
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
import fr.mesabloo.heavymachdefense.MainGame

fun main() {
    val config = Lwjgl3ApplicationConfiguration()
    config.setIdleFPS(30)
    config.setForegroundFPS(60)
    config.setWindowedMode(768, 1024)
    config.setTitle("Heavy MACH: Defense")
    config.setWindowIcon(Files.FileType.Internal, "gfx/nIcon57.png")

    Lwjgl3Application(MainGame(), config)
}